FIAT/brezzi_douglas_marini.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, ref_el, degree, variant, interpolant_deg):
                 entity_ids[dim][entity] = []
 
         if variant == "integral":
-            facet = ref_el.get_facet_element()
```

Comment:
What's the difference here?  Is it broken for general complexes?

Reply:
Maybe so.  Isn't it just instantiating the correct reference element of cxdimension 1?

Reply:
Right, I can enable that.

Reply:
I am going to add `SimplicialComplex.get_facet_element()`. But I will not use it in the element definitions, as I need to call `ref_el.construct_subelement(dim)` for a generic `dim`.